tests/python/contrib/test_ethosu/test_networks.py

```diff
@@ -142,18 +141,8 @@ def test_networks_with_usmp_and_cascader_wo_striping(accel_type, model_url, work
 @pytest.mark.parametrize(
     "accel_type, model_url, workspace_size",
     [
-        # Checks the same test case multiple times to make sure its not flaky
-        ("ethos-u55-256", MOBILENET_V1_URL, 1010000),
-        ("ethos-u55-256", MOBILENET_V1_URL, 1010000),
-        ("ethos-u55-256", MOBILENET_V1_URL, 1010000),
-        ("ethos-u55-256", MOBILENET_V1_URL, 1010000),
-        ("ethos-u55-256", MOBILENET_V1_URL, 1010000),
-        # Checks the same test case multiple times to make sure its not flaky
-        ("ethos-u55-256", MOBILENET_V2_URL, 1400000),
-        ("ethos-u55-256", MOBILENET_V2_URL, 1400000),
-        ("ethos-u55-256", MOBILENET_V2_URL, 1400000),
-        ("ethos-u55-256", MOBILENET_V2_URL, 1400000),
-        ("ethos-u55-256", MOBILENET_V2_URL, 1400000),
+        ("ethos-u55-256", MOBILENET_V1_URL, 1005312),
+        ("ethos-u55-256", MOBILENET_V2_URL, 1162368),
```

Comment:
Maybe revert to old values in test since cascade scheduling is unstable, I've prepared a fix for this.

Reply:
Looks like I can confirm instability with the cascade scheduling as in two subsequent runs of `tests/python/contrib/test_ethosu/test_networks.py::test_networks_with_usmp_and_cascader_with_striping[ethos-u55-256-model_url1-...` the _hill_climb_ allocator was supplied at first run with 8 and, at second, with 7 buffers to allocate, which resulted in different allocated sizes (1374688 and 1162448 bytes correspondingly, in both cases all the buffers "conflicting" with each other).

Reply:
url1 = MOBILENET_V2_URL
@Alexey-Yazev please advise with the pool size values as currently all Cortex-M tests failed

Reply:
As I see the test_networks.py passed, there are problems with the test_codegen.py. There are cases when buffer_info_arr (in HillClimbAllocator::PlanMemory) is empty, I suggest adding a size check for buffer_info_arr and return empty map in this case.